ultravox/inference/infer.py

```diff
@@ -68,55 +88,53 @@ def infer(
         output_tokens = output.sequences[0][input_len:]
         output_text = self.tokenizer.decode(output_tokens, skip_special_tokens=True)
```

Comment:
I still see `<|eot_id|>` is gradio chat window, e.g.,
```
It's nice to meet you. Is there something I can help you with or would you like to chat?<|eot_id|>
```

Reply:
I see this as well, but I see that even with non-streaming inference. I think there's a separate problem with this code in `_dataproc`: @farzadab 
```
      if "<|eot_id|>" in self.tokenizer.added_tokens_encoder:
            terminators.append(self.tokenizer.convert_tokens_to_ids("<|eot_id|>"))
```

Reply:
maybe we're not setting `added_tokens_encoder` anymore?

Reply:
re `<|eot_id|>` in `terminators`: I don't think having the terminator means it shouldn't be in the output.
I assumed the way we used to remove that was with `skip_special_tokens=True`. Not sure if that behaviour is working as expected now or not.

Reply:
Fixed in the latest rev.